enterprise_access/apps/content_assignments/tests/test_utils.py

```diff
@@ -192,3 +236,41 @@ def test_get_self_paced_normalized_start_date_self_paced(self, start_date, end_d
         else:
             assert get_self_paced_normalized_start_date(start_date, end_date, course_metadata) == \
                    start_date
+
+    @ddt.data(
+        # Expects the content_key for an is_assigned_course_run
```

Comment:
[nit/clarification] What is meant by `"for an is_assigned_course_run"`? Perhaps a minor revision to:

```suggestion
        # Expects course run associated with configured content_key for run-based assignment
```